packages/domain-search/src/components/index.ts

```diff
@@ -0,0 +1 @@
+export { Filter } from './search-bar/filter';
```

Comment:
We shouldn't export the component (which is tied to page-specific logic). Let's export only dumb components (i.e., the one from `ui`, which we're already doing).

In other words, let's use the existing logic within the Calypso implementation and only use the presentational filter button and popover components there, with all the logic juggling for the old Calypso version still happening there.

Reply:
Yeah, I wasn't sure about this. I think it makes sense to only export the dumb components, thanks for the clarification.